common/utils.go

```diff
@@ -75,12 +75,11 @@ func SetupLOG(logger *l.Logger, logLevel string) {
 	switch os.Getenv("INITSYSTEM") {
 	case "SYSTEMD":
 		fileFormat = l.MustStringFormatter(
-			`%{level:.4s} %{id:03x} %{shortfunc} > %{message}`)
+			`%{level:.4s} %{message}`)
```

Comment:
`callpath` is even more verbose. The debugging output is currently very hard to use because of all the noise. If you want to have all that noise we should add it behind an option in a new PR. This PR is an immediate resolution to the problem of noise in even normal warning messages.